.github/workflows/ci.yaml

```
@@ -0,0 +1,50 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches:
+      - "**"
+    tags:
+      - "**"
```

If you want all branches/tags, you can just leave these out:

```suggestion
on:
  push:
```